README.md

```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
 To contribute follow the following steps.
 
 1. Fork the repository
-2. open the repository root
-3. Install ionic globally by running `npm i @ionic/cli -g` (you may need to install node if you don't have it installed already)
-4. Run `yarn` or `npm i`
-5. Run `yarn start`
-6. App should be served in the browser!
+2. Open the repository root
+3. Download and install node at https://nodejs.org/en/download/ (If not already installed)
+4. Open nodejs directory
+5. Install ionic globally by running `npm i ionic -g`
```

Comment:
Seems like you resolved conflicts incorrectly because you overwrote this change: https://github.com/ShacharHarshuv/open-ear/commit/cf4ec538762e2b41b1851f238aac39d1573e209c

Reply:
Fixed